person.rb

```diff
@@ -23,3 +32,11 @@ def generate_id
     rand(1..1000)
   end
 end
+
+person = Person.new(name: 'maximilianus', age: 25)
+capitalized_person = CapitalizeDecorator.new(person)
+trimmed_person = TrimmerDecorator.new(capitalized_person)
+
+puts person.correct_name
+puts capitalized_person.correct_name
+puts trimmed_person.correct_name
```

Comment:
- [ ] [optional] It would be better professionally and considered good practice to add all the test code you wanted inside a separate test file so kindly remove this code and include it inside a test file.

Reply:
@Omar-Muhamad thank you for the heads-up. Much appreciated.